test/main.js

```diff
@@ -30,16 +31,6 @@ test('stdin-filename option with stdin', async t => {
 	t.regex(stdout, /unicorn-file:/);
 });
 
-test('reporter option', async t => {
-	const filepath = await tempWrite('console.log()\n', 'x.js');
-
-	try {
-		await main(['--reporter=compact', filepath]);
-	} catch (err) {
-		t.true(err.stdout.indexOf('Error - ') !== -1);
-	}
-});
```

Comment:
Sorry, I misread this code. It should not be removed. It's not using the `compact` option.